OpenRA.Mods.Common/AI/BotModules/HarvesterBotModule.cs

```diff
@@ -20,51 +20,68 @@
 
 namespace OpenRA.Mods.Common.AI
 {
-	class AIHarvesterManager
+	[Desc("Manages bot harvesters to ensure they always continue harvesting as long as there's resources on the map.")]
+	public class HarvesterBotModuleInfo : IBotModuleInfo
```

Comment:
Would there be any value in making these conditional?

Reply:
Can't say at this point, then again it's not much work so I might as well go for it now and potentially save us a PR down the line.